types/redux-first-router/index.d.ts

```diff
@@ -152,9 +183,15 @@ export interface Options {
     onAfterChange?<S>(dispatch: Dispatch<S>, getState: StateGetter): void;
     onBackNext?<S>(dispatch: Dispatch<S>, getState: StateGetter): void;
     restoreScroll?(history: History): ScrollBehavior;
+    initialDispatch?: boolean;
+    querySerializer?: QuerySerializer;
     navigators?: NavigatorsConfig;
 }
 
+// Params and Payload differ from Flow definition, as they can be arbitrarily shaped
+export type Params = any;
+export type Payload = any;
+
```

Comment:
whatsup guys. I'm excited to see this, but I won't have time to look it over for a few days. I'm also not a typescript expert. Just ping me if you have any precise questions. ...i will say that starting tomorrow I'll be working on RFR full time for a week or for however long it takes to complete the features necessary to make it feature complete. Mainly small things, but one big new feature: pre-fetching. Here's the list:

- prefetching (both thunks + async-fetched chunks)
- replace reducers for code splitting
- blocking page leaving with a confirmation dialog
- pathless routes (for using RFR style routesMaps which people like for actions that dont need paths)
- minor enhancements to `to/fromPath`
- react navigation docs and maybe a few new things

And honestly a few other small things. It won't take me long and it's all I'll be doing--but basically just know that the typings will need to update in no longer than 2 weeks. At that point, I'd be inclined to spend time nailing this with you guys. Iteration is key though, so dont let what im saying stop u. Just wanted to let you know.